docs/contributing/how-to-make-a-reproducible-test-case.md

```diff
@@ -15,22 +15,48 @@ The [most important part of any bug report](https://developer.mozilla.org/en-US/
 A reproducible test case is a great way to share a specific environment that causes a bug. Your reproducible test case is the best way to help people that want to help _you_.
 
 ## Steps to create a reproducible test case
-
-- Create a new Gatsby site with a starter, the official `gatsby-starter-minimal` starter is a great 'barebones' starting point here: `gatsby new bug-repro https://github.com/gatsbyjs/gatsby-starter-minimal`
-- Add any Gatsby plugins that relate to the issue. For example, if you're having problems with Gatsby MDX you should install and configure [`gatsby-plugin-mdx`](/plugins/gatsby-plugin-mdx/). Only add plugins that are needed to demonstrate the problem.
+- Install Gatsby CLI if you haven’t using this [guide](https://www.gatsbyjs.com/docs/reference/gatsby-cli/).
+- Create a new Gatsby site with a starter, the official `gatsby-starter-minimal` starter is a great 'barebones' starting point here: `gatsby new bug-repro https://github.com/gatsbyjs/gatsby-starter-minimal` There are other Gatsby starters you can use [here](https://www.gatsbyjs.com/starters/)
+- Add any Gatsby plugins that relate to the issue. For example, if you're having problems with Gatsby MDX you should install and configure [`gatsby-plugin-mdx`](/plugins/gatsby-plugin-mdx/) in the directory of your site (i.e bug-repro). Remember to update your `gatsby-config.js` with the plugin you installed and only add plugins that are needed to demonstrate the problem.
 - Add the code needed to recreate the error you've seen.
-- Publish the code (your GitHub account is a good place to do this) and then link to it when [creating an issue](/contributing/how-to-file-an-issue/).
-
-## Online Development Environments
```

Comment:
Why did you remove this?

Reply:
We've always encouraged customers to reproduce issues locally and I thought this may not be needed.

Reply:
This way they can reproduce it locally, it's just running inside a VM, not on their local computer so we're fine with those online IDEs